.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
+
+language: java
+
+jdk:
+  - oraclejdk8
+
+before_install:
+  - chmod +x gradlew
+
+script:
+  - ./gradlew test
```

Comment:
is this required now? since it is a maven project we don't have the gradlew file anymore